.github/workflows/android.yml

```diff
@@ -0,0 +1,52 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+    paths: kotlin/**
+  pull_request:
```

Comment:
Do you also need to add the `paths` here so these tasks are only ran for the `kotlin` directory or does it inherit the `push` settings?

Reply:
Oh you just added it.